service/src/main/java/bio/terra/landingzone/stairway/flight/create/CreateAzureLandingZoneStep.java

```diff
@@ -72,7 +73,8 @@ public StepResult doStep(FlightContext context) throws RetryException {
       return new StepResult(StepStatus.STEP_RESULT_FAILURE_FATAL, e);
     } catch (Exception e) {
       // TODO SG: check if we can retry?
-      return new StepResult(StepStatus.STEP_RESULT_FAILURE_FATAL, e);
+      return new StepResult(
+          StepStatus.STEP_RESULT_FAILURE_FATAL, new FlightExceptionTranslator(e).translate());
```

Comment:
Does this mean the original exception is lost? I think that for troubleshooting it should be at least logged.

Reply:
Added logging